forward43/setup_elasticsearch.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import pandas as pd
+
+import utils.elasticsearch as es_utils
+
+
+def create_index_with_mapping(es):
+
+    # FIXME: The mappings should be retrieved from the mappings PR
+    mappings = {
+        "properties": {
+            "name": {
+                "type": "keyword"
+            },
+            "creator": {
+                "type": "text"
+            },
+            "pledged": {
+                "type": "float"
+            },
+            "state": {
+                "type": "text"
+            }
+        }
+    }
+
+    is_created = es_utils.create_index(
+        es_object  = es, 
+        index_name = 'forward43', 
+        mappings   = mappings,
+        n_shards   = 1,
+        n_replicas = 0
+    )
+
+
+def read_json_data(es):
```

Comment:
Some naming convention notes: `read_json_data` makes it sound like the function will read json data and return it, which another function (such as `load_to_es()`) will use to load the data.
The way this is setup has a combined role, which is frowned upon code-standard-wise, but ok given the size right now, but I'd rename it to read_and_load_data()